test/donations.routes.test.js

```diff
@@ -29,7 +30,7 @@ const stripeMock = require('./mocks/stripe');
 /**
  * Tests.
  */
-describe('payments.routes.test.js', () => {
+describe('donations.routes.test.js', () => {
```

Comment:
Yes feel free to leave it as is. I had a trello task for it. Will move it
to GitHub.
On Tue, Apr 26, 2016 at 10:17 AM Sébastien Dubois <notifications@github.com>
wrote:

> In test/donations.routes.test.js
> <https://github.com/OpenCollective/api/pull/234#discussion_r61093171>:
>
> > @@ -29,7 +30,7 @@ const stripeMock = require('./mocks/stripe');
> >  /**
> >   * Tests.
> >   */
> > -describe('payments.routes.test.js', () => {
> > +describe('donations.routes.test.js', () => {
>
> So it's in your hands to update the website side?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenCollective/api/pull/234/files/29ec3f3523c35d7769909856960cb9da062cb8f0#r61093171>
>